include/zephyr/dt-bindings/clock/mcux_lpc_syscon_clock.h

```diff
@@ -74,4 +74,6 @@
 #define MCUX_PORT4_CLK			MCUX_LPC_CLK_ID(0x0C, 0x04)
 #define MCUX_PORT5_CLK			MCUX_LPC_CLK_ID(0x0C, 0x05)
 
+#define MCUX_ENET_QOS_CLK		42
```

Comment:
Can we align this with the `MCUX_LPC_CLK_ID` scheme used for other clock IDs?

Reply:
hmmmm.... I thought I had done this, weird